### PR TITLE
[ioctl] add checking --insecure

### PR DESCRIPTION
**Pull request by huof6829**
# Description
check `--insecure` value  
If has in cmdline, read the value then connect grpc
if not has in cmdline, read from config file

Fixes  https://github.com/iotexproject/iotex-core/issues/3716

## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Code refactor or improvement
- [] Breaking change (fix or feature that would cause a new or changed behavior of existing functionality)
- [] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [] make test
- [] fullsync
- [] Other test (please specify)

**Test Configuration**:
- Firmware version:
- Hardware:
- Toolchain:
- SDK:

# Checklist:
- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes
- [] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by huof6829** [comment]
test : 
截屏2023-01-12 19 33 30

**Review by CoderZhi — COMMENTED** [review]
can we change lines like https://github.com/iotexproject/iotex-core/pull/3765/files#diff-480b6a42e77da9d54219c1fb6119137e050dfd87134a5d041c80e2ff978b7461L86 as
`AccountCmd.PersistentFlags().BoolVar(&config.Insecure, "insecure", config.ReadConfig.SecureConnect, config.TranslateInLang(_flagInsecure, config.UILanguage))`?

**Comment by huof6829** [comment]
> can we change lines like https://github.com/iotexproject/iotex-core/pull/3765/files#diff-480b6a42e77da9d54219c1fb6119137e050dfd87134a5d041c80e2ff978b7461L86 as `AccountCmd.PersistentFlags().BoolVar(&config.Insecure, "insecure", config.ReadConfig.SecureConnect, config.TranslateInLang(_flagInsecure, config.UILanguage))`?

OK.  test right after modified

File: ioctl/cmd/account/account.go [hunk 1]
```diff
@@ -192,10 +193,11 @@ func GetAccountMeta(addr string) (*iotextypes.AccountMeta, error) {
 		ctx = metautils.NiceMD(jwtMD).ToOutgoing(ctx)
 	}
 	response, err := cli.GetAccount(ctx, &request)
-
 	if err != nil {
-		sta, ok := status.FromError(err)
-		if ok {
+		if sta, ok := status.FromError(err); ok {
+			if sta.Code() == codes.Unavailable {
+				return nil, output.NewError(output.APIError, "check endpoint or secureConnect in ~/.config/ioctl/default/config.default or cmd flag value if has", nil)
```
Comment (dustinxie) [inline ioctl/cmd/account/account.go:199]: or if --insecure flag value is properly set on command line

**Review by dustinxie — APPROVED** [review]

**Review by millken — APPROVED** [review]

**Comment by raullenchai** [comment]
Close b/c it has been inactive for 1+yr.

**Pull request closed without merging**